### DISCUSSION TITLE
Include Total Row data in Downloaded CSV report

### DISCUSSION THREAD

**Discussion by Sandy381** [fbaligand/kibana-enhanced-table#310] [General]
Hi @fbaligand ,

When we use Sum, Average etc option on enhanced table. 

We can see Total label and values on display screen.

But when we click on Download buttons "Raw" & "Formatted" , the total row is not present in CSV.

![Screenshot 2023-02-10 at 12 09 00 PM](https://user-images.githubusercontent.com/74658016/218020979-24765238-00a1-4cd0-b505-19b7023502bb.png)

It is not compulsary, but it would be helpful if we have this Total Row in Downloaded CSV as well.

**Comment by fbaligand** [comment]
Yes, this is default behavior that you can see also in standard Data Table.
But if you go to Options tab, there is an option to add total line in CSV export.
